app/react/Layout/TableRows.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+
+import { connect } from 'react-redux';
+import { TableRow } from 'app/Library/components/TableRow';
+import { IStore, TableViewColumn } from 'app/istore';
+import { IImmutable } from 'shared/types/Immutable';
+import { EntitySchema } from 'shared/types/entityType';
+import Immutable from 'immutable';
+
+interface TableRowsProps {
+  columns: TableViewColumn[];
+  storeKey: 'library' | 'uploads';
+  clickOnDocument: (e: React.SyntheticEvent, doc: EntitySchema, active: boolean) => any;
+  onEndScroll: (amount: number, from: number) => void;
+  documents: IImmutable<{ rows: EntitySchema[] }>;
+  rowListZoomLevel: number;
+}
+
+const defaultProps = {
+  documents: Immutable.fromJS({ rows: [] }),
+  rowListZoomLevel: 2,
+};
+
+class TableRowsComponent extends Component<TableRowsProps> {
+  static defaultProps = defaultProps;
+
+  render() {
+    console.log('render rows');
```

Comment:
Please, remove this!!!!

Reply:
I believe that our lint is not picking up TSX files: the previous PR that had index used as keys should have been picked up as well.  It would be good if you, that already did some work on the linters, could double check that, please.